gnovm/stdlibs/testing/testing.gno

```diff
@@ -39,6 +39,93 @@ func Recover(result Setter) {
 	panic(r)
 }
 
+func extractPanicErr(r interface{}) string {
```

Comment:
I love this PR, except this part, again. Sorry, but I need to mark as blocked for this.

Reply:
Swapped:

[b48532e](https://github.com/gnolang/gno/pull/2475/commits/b48532e94b7c64ea77db2ba882c9a2f15a1ec607)